package.json

```diff
@@ -141,7 +141,7 @@
     "webextension-polyfill": "0.7.0"
   },
   "engines": {
-    "node": ">=14.15.0",
+    "node": ">=14.15.0 <17",
```

Comment:
for some reason latest LTS (18) fails to build this. I did not investigate, but added check to prevent this.

Reply:
+1. a lot of our packages and their dependencies are out of date, and there have been a lot of changes made w.r.t. esm support, so we've been in a tough spot. However, you can see our support goals at https://github.com/ipfs/ipfs-gui/tree/SgtPooki-version-support-definition#nodejs (We can link to main branch when https://github.com/ipfs/ipfs-gui/pull/97 gets merged)